src/main/java/core/basesyntax/model/User.java

```diff
@@ -1,10 +1,26 @@
 package core.basesyntax.model;
 
 import java.util.List;
-
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String username;
+    @OneToMany(fetch = FetchType.EAGER,
```

Comment:
Using fetch type `EAGER` is a bad practice for toMany relations. (For toOne relations too to be honest, you'll later see why).